api/v1alpha1/nonadminbackup_types.go

```diff
@@ -32,14 +32,12 @@ type NonAdminBackupSpec struct {
 
 	// BackupSpec defines the specification for a Velero backup.
 	BackupSpec *velerov1api.BackupSpec `json:"backupSpec,omitempty"`
-
-	// BackupStatus captures the current status of a Velero backup.
-	BackupStatus *velerov1api.BackupStatus `json:"backupStatus,omitempty"`
 }
 
 // NonAdminBackupStatus defines the observed state of NonAdminBackup
 type NonAdminBackupStatus struct {
```

Comment:
Non-Admin backup status can have more details than velero backup status, eg: validation status or reconciliation status of the non-admin backup CR itself.

Reply:
Modified. We now have (two examples from this PR):
```
spec: {}
status:
  failureReason: NonAdminBackup CR does not contain valid VeleroBackupSpec
  phase: FailedValidation
```


```
spec:
  backupSpec:
    volumeSnapshotLocations:
      - velero-sample-1
    defaultVolumesToFsBackup: false
    csiSnapshotTimeout: 10m0s
    ttl: 720h0m0s
    itemOperationTimeout: 4h0m0s
    metadata: {}
    storageLocation: velero-sample-1
    hooks: {}
    snapshotMoveData: false
status:
  backupStatus:
    expiration: '2024-04-05T16:46:15Z'
    failureReason: >-
      unable to get credentials: unable to get key for secret: Secret
      "cloud-credentials" not found
    formatVersion: 1.1.0
    phase: Failed
    startTimestamp: '2024-03-06T16:46:15Z'
    version: 1
```